serialization.md

```diff
@@ -1,10 +1,7 @@
-# Docker Image Specification v1.0.0
```

Comment:
Unfortunately, this specification is fairly out of date after the Docker 1.10 changes. Most of these changes are documented [here](https://gist.github.com/aaronlehmann/b42a2eaf633fc949f93b). We are going to update this to correspond with the current implementation described in the referenced document, which can be used as a more up to date base.

Given that, it seems prudent to keep this PR focused on the manifest specification.